migrations/versions/1330_add_suppliers_active_flag.py

```diff
@@ -0,0 +1,34 @@
+"""add suppliers active flag
+
+Adds a new column to the suppliers table, 'active', that indicates whether a
+supplier is currently valid.
+
+Revision ID: 1330
+Revises: 1320
+Create Date: 2019-06-26 11:14:56.085586
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+
+# revision identifiers, used by Alembic.
+revision = '1330'
+down_revision = '1320'
+
+
+def upgrade():
+    # We want this column to be DEFAULT true and NOT NULLABLE, but creating a column
+    # with defaults is a slow operation, so we split it up a bit.
+    op.add_column('suppliers', sa.Column('active', sa.Boolean(), nullable=True))
+    op.alter_column('suppliers', 'active', server_default=sa.text('true'))
+
+    # Set any NULL values
+    # NULLs are active suppliers (i.e. they have not been made inactive)
+    op.execute("UPDATE suppliers SET active = true WHERE active = NULL")
```

Comment:
To be useful this has to be done in a separate transaction (i.e. migration)

Reply:
I experimented with this but it seems that alembic puts all migrations applied into one transaction anyway?

Reply:
Okay I've added your patch and split the migration in two.